helm_deploy/prisoner-content-hub-backend/values.yaml

```diff
@@ -14,11 +14,11 @@ tier: backend
 application:
   port: 8080
   liveness:
-    endpoint: /api/health
+    endpoint: /
```

Comment:
Note, these will be moved to `/health` once the change is in production. Needs to be set to this otherwise the pods never successfully pass a health check as the URL they check doesn't exist until the module has been removed and added back in the Drupal admin area.